Box.V2/Config/Constants.cs

```diff
@@ -220,11 +220,59 @@ public static class RequestParameters
             public const string Pending = "pending";
 
             public const string IfMatch = "If-Match";
+
+            /*** Values for specifically representations endpoint ***/
+            public const string XRepHints = "x-rep-hints";
+            public const string SetContentDispositionType = "set_content_disposition_type";
+            public const string SetContentDispositionFilename = "set_content_disposition_filename";
+            public const string RepresentationField = "representations";
+            public const int RepresentationRequestRetryTime = 3000;
```

Comment:
make it private inside the FilesManager, since this is only related to the retry but not the request